src/instructlab/model/chat.py

```diff
@@ -61,6 +62,30 @@
 
 PROMPT_PREFIX = ">>> "
 
+DEFAULT_ENDPOINT = "http://127.0.0.1:8000/v1"
+
+
+def is_openai_server_and_serving_model(
+    endpoint: str, api_key: str, http_params: HttpClientParams
+) -> bool:
+    """
+    Given an endpoint, returns whether or not the given response is expected
```

Comment:
`given response` which response is it?

Reply:
The response from the endpoint.

Reply:
When you refer to "the given response" I read that as if it's another arg - is it `http_params`?

Reply:
No - those are just values that we need to be able to make the request using the OpenAI client.

Reply:
@nathan-weinberg This is just saying that we received a response consistent with that of an OpenAI-compatible server. Consider the case when we have another REST server at the URL but is completely unrelated - the response we get wouldn't be expected.

Reply:
gotcha @RobotSail - might i suggest the following `Given an server endpoint, returns whether or not the server is OpenAI-compatable`

@booxter wdyt?

Reply:
Yes Nathan, that wording is much better. Lets definitely change it since the way I wrote it isn't very clear.

Reply:
@nathan-weinberg this is definitely better. The function also checks that models are served though. Maybe it should have been also captured. (That said, the name of the function is so clear that I don't know if there's much of value in having any docstring...)